addon/ng2/blueprints/ng2/files/src/tsconfig.json

```diff
@@ -8,9 +8,13 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noImplicitAny": false,
+    "outDir": "../dist/",
     "rootDir": ".",
     "sourceMap": true,
     "sourceRoot": "/",
     "target": "es5"
-  }
+  },
+  "exclude": [
+    "../typings/main.d.ts"
```

Comment:
Excluding the file means the typings we have will never be accessible.

This is the problem I ran into: typescript allows you to either list all files, or exclude some of them. To use exclude you'd need typings to be installed on the `src/` folder, and exclude the ones you don't want.